README.md

```diff
@@ -26,8 +26,12 @@ Simple stock API for fetching information related to stocks.
 - Execute the command: `./mvnw clean package`
 - Run the following from the root: `echo ENV POLYGON_API_KEY="<INSERT_YOUR_API_KEY>" >> Dockerfile`
 - Run the following: `docker build -t <INSERT_WHAT_YOU_WANT_TO_CALL_THIS_IMAGE> .`
-- Run the following: `docker run -p 8080:8080 -t <THE_NAME_YOU_MADE_UP_IN_THE_PREVIOUS_COMMAND>`
+- Run the following: `docker run -e POLYGON_API_KEY=<your key> -p 8080:8080 -t <THE_NAME_YOU_MADE_UP_IN_THE_PREVIOUS_COMMAND>`
```

Comment:
For whatever reason I could not get this `--env` flag to pass my environment variables to the docker container. Probably just a docker rookie mistake.

Reply:
Any ideas what I could have been doing wrong @ahplummer??

Reply:
I'm on Linux, so it may be my docker runtime allowing for the -e passing in.  You're using just `-e`, right?  What is your host? Mac, Windows?  I believe on Mac, `-e` also works. Unsure on Windows; I haven't used Windows in quite a few years.

Reply:
Hey there: [Some thread](https://stackoverflow.com/questions/43049337/docker-for-windows-passing-env-variable-to-cmd-in-dockerfile-doesnt-work) indicates that non-POSIX systems like Windows need a bit of hand-holding on that command line switch....

Reply:
Right. I am using Windows. I have WSL but there is probably a step I am missing to utilize it.